### DISCUSSION TITLE
Disabling notifications relating to easylist/easyprivacy/ublockfilters on startup

### DISCUSSION THREAD

**Discussion by ceuk** [Jelmerro/Vieb#550] [Q&A] [answered]
Is there any way to disable the 6 notifications that fire every time you start the browser (when `adblocker` is set to `update`)?

**Comment by Jelmerro** [comment]
Currently no, but I'll think of setting for this, possibly a generic one to control notifications more broadly. Naming suggestions are welcome.

**Reply by Jelmerro** [reply, marked as answer]
Implemented in 12.0.0 :tada: see `:h adblockernotifications` for help.
